Gemfile

```diff
@@ -13,7 +13,7 @@ group :development do
   gem 'komenda', '~> 0.1.6'
   gem 'launchy'
   gem 'rake', '10.4.2'
-  gem 'rubocop'
+  gem 'rubocop', '~> 0.43.0'
```

Comment:
@hferentschik , Do you mean everytime, I need to remove Gemfile.lock file to map to ci like bundler environment?